application-xpoll-ui/src/main/resources/XPoll/XPollJSExtension.xml

```diff
@@ -151,6 +151,22 @@
         }
       });
     });
+    $('.xpollSelect').change(function() {
+      var changedElement = $(this);
+      $('.xpollSelect').each(function() {
+        if ($(this).attr('proposal') != changedElement.attr('proposal') &amp;&amp; $(this).val() == changedElement.val())
+          $(this).find('option:eq(0)').prop('selected', true);
+      });
+
+      var hiddenField = '#option' + $(this).val();
+      var proposal = $(this).attr('proposal');
+      $('.xpollHiddenInput').each(function() {
+        if ($(this).val() == proposal)
```

Comment:
You should always add the `{}`, even if there is only a line of code inside. Other lines could be added after, not necessarily by you, and someone could not observe that the brackets are missing